src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +145,56 @@ $(document).ready(() => {
     $('#btn-collapse-all-students').on('click', () => {
         adminSearchCollapseAllStudents();
     });
+
+    $('.openEmailApplicationDefaultValues').submit((e) => {
+        e.preventDefault();
+        const $entry = $(e.target);
+        const studentEmail = $entry.parentsUntil('.fslink_student').find('input[name=studentEmail]').val();
+        const subjectType = $entry.find('input[name=subjectType]').val();
+        const courseName = $entry.find('input[name=courseName]').val();
+        const courseId = $entry.find('input[name=courseId]').val();
+        const studentName = $entry.find('input[name=studentName]').val();
+        const sessionStatus = $entry.find('input[name=sessionStatus]').val();
+        const sessionName = $entry.find('#feedback-session-name').text();
+        const sessionNameWithoutDate = sessionName.substring(0, sessionName.lastIndexOf(' ['));
+
+        const relatedLink = encodeURIComponent($entry.find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+        const supportEmail = `${Const.TEAMMATES_EMAIL_ADDRESS}`;
+
+        const bodyHeader = `Hello ${studentName},`;
+        const bodyFooter =
+                    '%0D%0A%0D%0AIf you encounter any problems when using the system,'
+                    + ` you can email TEAMMATES support team at ${supportEmail}.`
+                    + '%0D%0A%0D%0ARegards,%0D%0ATEAMMATES Team.';
```

Comment:
Use 8 spaces from the previous line (ie. remove another 4 more spaces here). The indentation for `bodyContent` below is fine as `bodyContent` is indented 1 level deeper due to the if/else conditionals.

Reply:
@chowyb corrected!